tests/consensus_tests/test_snapshot_recovery_kill.py

```
@@ -0,0 +1,102 @@
+    assert_http_ok(r)
+    snapshot_url = f"{peer_urls[0]}/collections/{COLLECTION}/snapshots/{r.json()['result']['name']}"
+
+    # Kill target peer
+    recovery_url = peer_urls[2]
+    recovery_dir = Path(peer_dirs[2])
+    recovery_port = processes[2].http_port
+
+    monitor = PartialMonitor(recovery_url)
+    monitor.start()
+
+    requests.put(f"{recovery_url}/collections/{COLLECTION}/snapshots/recover?wait=false", json={"location": snapshot_url})
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)